src/sentry/features/manager.py

```diff
@@ -178,6 +181,9 @@ def add(
             feature_option_name = f"{FLAGPOLE_OPTION_PREFIX}.{name}"
             register(feature_option_name, type=Dict, default={}, flags=FLAG_AUTOMATOR_MODIFIABLE)
 
+        if default is not None:
+            settings.SENTRY_FEATURES[name] = default
```

Comment:
How did it get this value before?

Reply:
It was set directly in `SENTRY_FEATURES`

Reply:
We would have to manually update `server.py` with the default. If no default was provided the default value would be `False` based on logic in `FeatureManager` https://github.com/getsentry/sentry/blob/88a8cbc633096d26da539157ff3edadc9c47e792/src/sentry/features/manager.py#L261

Reply:
oh duh, I read this backwards lol

Reply:
One nice thing about having this in the server file is that it was somewhat documenting for on-prem users. But I'm sure not many people actually looked there.